homework_5/task_1.py

```diff
@@ -0,0 +1,18 @@
+while True:
+    n = int(input("Enter n [1/49]: "))
+    if n > 0 and n < 50:
+        break
+
+for i in range(1, n + 1):
+    if i < 10:                          # Code from ln.7 to ln.10 does absolutely nothing 
+        print('', i, " - ", end="")     # but I implemented This part just for better readability 
```

Comment:
ასე ხომ არ უნდა იყოს? 
```suggestion
        print(' ', i, " - ", end="")     # but I implemented This part just for better readability 
```